--- a/Makefile.am
+++ b/Makefile.am
@@ -1,3 +1,3 @@
 ACLOCAL_AMFLAGS = -I m4 -Wnone
 
-SUBDIRS = gpslogger omf_nmetrics omf_trace otg wlanconfig simple ripwave_monitor
+SUBDIRS = gpslogger otg wlanconfig simple ripwave_monitor
--- a/configure.ac
+++ b/configure.ac
@@ -16,7 +16,7 @@ AC_CONFIG_FILES([Makefile
 		simple/Makefile
 		])
 
-AC_CONFIG_SUBDIRS([gpslogger omf_nmetrics omf_trace otg ripwave_monitor wlanconfig])
+AC_CONFIG_SUBDIRS([gpslogger otg ripwave_monitor wlanconfig])
 
 # Generate output files
 AC_OUTPUT
