Index: oml2-apps-2.8.0/Makefile.am
===================================================================
--- oml2-apps-2.8.0.orig/Makefile.am	2012-07-17 10:31:21.000000000 +0200
+++ oml2-apps-2.8.0/Makefile.am	2012-07-18 09:14:40.000000000 +0200
@@ -1,3 +1,4 @@
 ACLOCAL_AMFLAGS = -Wall
 
-SUBDIRS = gpslogger nmetrics iperf otg2 ripwavemon trace wlanconfig 
+SUBDIRS = gpslogger nmetrics otg2 wlanconfig ripwavemon
+
Index: oml2-apps-2.8.0/configure.ac
===================================================================
--- oml2-apps-2.8.0.orig/configure.ac	2012-07-17 10:31:21.000000000 +0200
+++ oml2-apps-2.8.0/configure.ac	2012-07-18 09:13:02.000000000 +0200
@@ -10,7 +10,7 @@
 AC_PROG_INSTALL
 AC_PROG_MAKE_SET
 
-AC_CONFIG_SUBDIRS([gpslogger iperf nmetrics otg2 ripwavemon trace wlanconfig])
+AC_CONFIG_SUBDIRS([gpslogger nmetrics otg2 ripwavemon wlanconfig])
 
 # Generate output files
 AC_CONFIG_FILES([Makefile])
