--- a/omf-resctl/ruby/omf-resctl/omf_driver/atheros.rb
+++ b/omf-resctl/ruby/omf-resctl/omf_driver/atheros.rb
@@ -84,9 +84,9 @@ class AtherosDevice < WirelessDevice
       when 'type'
         # 'value' defines type of operation
         type = case
-          when value == 'a' : 1
-          when value == 'b' : 2
-          when value == 'g' : 3
+          when value == 'a' then 1
+          when value == 'b' then 2
+          when value == 'g' then 3
           else
             raise "Unknown type. Should be 'a', 'b', or 'g'."
         end
@@ -94,16 +94,16 @@ class AtherosDevice < WirelessDevice
 
       when "mode"
         mode = case
-          when value == 'master' : 'ap'
-          when value == 'Master' : 'ap'
-          when value == 'managed' : 'sta'
-          when value == 'Managed' : 'sta'
-          when value == 'ad-hoc' : 'adhoc'
-          when value == 'Ad-Hoc' : 'adhoc'
-          when value == 'adhoc' : 'adhoc'
-          when value == 'AdHoc' : 'adhoc'
-          when value == 'monitor' : 'monitor'
-          when value == 'Monitor' : 'monitor'
+          when value == 'master' then 'ap'
+          when value == 'Master' then 'ap'
+          when value == 'managed' then 'sta'
+          when value == 'Managed' then 'sta'
+          when value == 'ad-hoc' then 'adhoc'
+          when value == 'Ad-Hoc' then 'adhoc'
+          when value == 'adhoc' then 'adhoc'
+          when value == 'AdHoc' then 'adhoc'
+          when value == 'monitor' then 'monitor'
+          when value == 'Monitor' then 'monitor'
           else
             raise "Unknown mode '#{value}'. Should be 'master', 'managed', or 'adhoc'."
         end
