--- a/omf-resctl/ruby/omf-resctl/omf_driver/ath5k.rb
+++ b/omf-resctl/ruby/omf-resctl/omf_driver/ath5k.rb
@@ -45,9 +45,9 @@ class Ath5kDevice < WirelessDevice
   def initialize(logicalName, deviceName)
     super(logicalName, deviceName)
     @driver = 'ath5k'
-    @wlanconfig = '/sbin/wlanconfig'
-    @iwconfig = '/sbin/iwconfig'
-    @iwpriv = '/sbin/iwpriv'
+    @wlanconfig = '/usr/sbin/wlanconfig'
+    @iwconfig = '/usr/sbin/iwconfig'
+    @iwpriv = '/usr/sbin/iwpriv'
   end
 
   #
